bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/io/CallCompletionStatusInterceptor.java

```diff
@@ -149,16 +152,16 @@ public void start(Listener<ResponseT> responseListener, Metadata.Headers headers
   }
 
   @Override
-  public <ReqT, RespT> CompletionStatusGatheringCall<ReqT, RespT> interceptCall(
-      MethodDescriptor<ReqT, RespT> methodDescriptor, Channel channel) {
-    return wrapCall(methodDescriptor, channel.newCall(methodDescriptor));
+  public <RequestT, ResponseT> CompletionStatusGatheringCall<RequestT, ResponseT> interceptCall(
+      MethodDescriptor<RequestT, ResponseT> method, CallOptions callOptions, Channel next) {
```

Comment:
That was the name that the gRPC team chose as the name, and I used eclipse to auto-generate the new signature.

https://github.com/grpc/grpc-java/blob/572f4332a75367e0d0e1cb096198839b68787670/core/src/main/java/io/grpc/ClientInterceptor.java#L61